### DISCUSSION TITLE
Comment in code fence

### DISCUSSION THREAD

**Discussion by carlba** [gsantner/markor#1555] [Q&A] [answered]
It seems like markdown content within code fences are still rendered as markdown.

The # List contents of logs folder line a rendered bigger than other lines when adding this to a markdown file and viewing it in the editor of Markor

```shell
# List contents of logs folder
ls -lah logs
```

**Comment by gsantner** [comment, marked as answer]
Highlighting of code blocks takes massive amount of cpu/mem, and so Markor not has exception for \`\`\` to guess if at that place other syntax should be disabled.
